min-turnup/phase1/azure/configure-vm.sh

```diff
@@ -0,0 +1,69 @@
+#! /bin/bash
+
+set -x
+set -o errexit
+set -o pipefail
+set -o nounset
+
+mkdir -p /etc/systemd/system/docker.service.d/
+cat <<EOF > /etc/systemd/system/docker.service.d/clear_mount_propagtion_flags.conf
+[Service]
+MountFlags=shared
+EOF
+cat <<EOF > /etc/systemd/system/docker.service.d/overlay.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
+EOF
+
+curl -sSL https://get.docker.com/ | sh
+
+apt-get update
+#apt-get dist-upgrade -y
+apt-get install -y jq
+
+systemctl start docker || true
+
+ROLE="node"
+if [[ $(hostname) = *master* ]]; then
+	ROLE="master"
+fi
+
+azure_file="/etc/kubernetes/azure.json"
+config_file="/etc/kubernetes/k8s_config.json"
+
+mkdir /etc/kubernetes
```

Comment:
mkdir /etc/kubernetes will fail if the directory already exists. Is this script supposed to be idempotent?